infer/src/istd/IStd.ml

```diff
@@ -5,10 +5,374 @@
  * LICENSE file in the root directory of this source tree.
  *)
 
-include Core_kernel
 
 [@@@warning "-32"]
 
+module Sys = struct
```

Comment:
Is that code copied from somewhere else? If so the copyright notices probably need changing. Maybe best to put them in their own file too (either all the modules in one file or one per module, not sure which is best).

Reply:
Most of the code here is copied from `Core` itself indeed, except for the completely trivial stubs (typically labelling functions). In any case, creating a separate function was on my TODO list, and I will adapt the copyright accordingly.

Reply:
Done, in a new file `Core_compat`.